setup.py

```diff
@@ -0,0 +1,38 @@
+import os
+from setuptools import setup
+
+"""
+Sets up the CLI and modules for `mach-perftest-notebook`.
+"""
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(this_directory, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
+with open(os.path.join(this_directory, "requirements.txt")) as f:
+    install_requires = f.read()
+
+setup(
+    name="mach-perftest-notebook",
+    version="0.0.1",
+    author="Gregory Mierzwinski",
+    author_email="gmierz1@live.ca",
+    description=(
+        "This tool will be used to standardize analysis/visualization techniques across Mozilla."
```

Comment:
nit: Should be `This tool is used to standardize analysis/visualization techniques across Mozilla.`